### ISSUE TITLE
I tried to migrate my project using this boilerplate to your newest version without immutable.js - getting many errors

### ISSUE THREAD

**Issue by SelfDevTV** [flexdinesh/react-redux-boilerplate#38]
Hi,

I have this project: https://github.com/SelfDevTV/forum-creator/tree/noImmutable

I have migrated everything to the newest version of this boilerplate (the one without immutable). 
But now when I start my app and I load it i see these errors in the browser console:

Errors: https://imgur.com/a/ilNGj2b (scroll down for more errors)

So I have now no clue what so ever what is going on.
I made exactly the same changes from the last commit. 

The component that constantly shows up in the dev tools is the "Navbar" - this is a container. 
Here are the files for this container:

https://github.com/SelfDevTV/forum-creator/tree/noImmutable/app/containers/Navbar

I think it has something to do with the newly added redux context.

When I `console.log` in the `injectReducer` the property `this.context` - I get back the context. 
When I `console.log` in the `injectSaga` the property `this.context` - the console.log doesn't even run. 

//EDIT: Just tried to just not render it, then the next container is in almost every error (Homepage). 
So it has to do something with the changed config. 
I would be very helpful if you can help me out here since it's big project I'm planning and I already have some code in it and I really don't want to use immutable :P

**I also can't really ask this on stackoverflow since I can't relate it to any specific part or file of the codebase.**

**Comment by SelfDevTV** [comment]
Oh my god that feels so great. I just diffed my repo with the boilerplate repo like this:

```
git remote add -f b path/to/repo_b.git
git remote update
git diff master remotes/b/master
git remote rm b
```

And I found out I made a single letter typo in the `injectSaga.js` file.

I typed: `static contextTypes = ReactReduxContext` instead of `static contextType = ReactReduxContext`

After I fixed that everything is back to normal and works perfectly. Man moments like this are why I love coding :D
